.circleci/config.yml

```
@@ -753,11 +753,11 @@ jobs:
       - setup_win
       - run:
           name: Build all windows CI targets
-          command: yarn bazel build --build_tag_filters=-ivy-only //packages/compiler-cli/... //tools/ts-api-guardian/...
+          command: yarn bazel build --build_tag_filters=-ivy-only-no-windows, //packages/compiler-cli/... //tools/ts-api-guardian/...
```

should there be a comma here?

```suggestion
          command: yarn bazel build --build_tag_filters=-ivy-only,-no-windows, //packages/compiler-cli/... //tools/ts-api-guardian/...
```

---

I fixed this - the comma was needed, but also we needed a `--build-tag-filters` option in the test command as well as the `--test_tag_filters` option.